src/misc/AttachShadowPolyfill.ts

```diff
@@ -11,17 +16,27 @@ export async function attachShadow(element: HTMLElement, init: ShadowRootInit &
   iframeBody.style.margin = '0';
   const shadowRoot = $$('div', { style: 'overflow: auto;' }).el;
   iframeBody.appendChild(shadowRoot);
-  autoUpdateHeight(iframe, shadowRoot);
-  if (init.mode === 'open') {
+  autoUpdateHeight(iframe, shadowRoot, options.onSizeChanged);
+  if (options.mode === 'open') {
     Object.defineProperty(element, 'shadowRoot', { get: () => shadowRoot });
   }
 
   return shadowRoot;
 }
 
-function autoUpdateHeight(elementToResize: HTMLElement, content: HTMLElement) {
+function autoUpdateHeight(elementToResize: HTMLElement, content: HTMLElement, onUpdate?: Function) {
+  let lastWidth = content.clientWidth;
+  let lastHeight = content.clientHeight;
+
   const heightObserver = new MutationObserver(() => {
+    if (lastWidth === content.clientWidth && lastHeight === content.clientHeight) {
+      return;
+    }
```

Comment:
If the `clientWidth` matches the original, then we return. However, if it does not, we resize below.

Should we then update `lastWidth` and `lastHeight`? Otherwise it seems like if the initial values are different, we will always try to resize.

Reply:
My bad, I somehow forgot...